Backend/Requests/addressGeocoding.py

```diff
@@ -0,0 +1,26 @@
+import requests
+
+# Nominatum API
+API_URL = 'https://nominatim.openstreetmap.org/search?'
+
+def geocoder(address):
+    params = {"q": address,
+            "format": 'json',
+            "limit": 1}
```

Comment:
Does the API only listen / react to Json parameters in the get request? Or would you also be able to send these in XML? Since R wants that the communication is based upon XML, sending the parameters in XML would also be an interesting Option.

Reply:
> Does the API only listen / react to Json parameters in the get request? Or would you also be able to send these in XML? Since R wants that the communication is based upon XML, sending the parameters in XML would also be an interesting Option.

These are URL parameters. The JSON is converted to URL-parameters by the get-function, they are not sent as the JSON which is quoted here. 
I also think communication in XML only concerns Backend to Frontend. This request is Backend -> Third party API

Reply:
Okay, just wanted to check that. Thanks for the detailed answer 👍